book/src/themes.md

```diff
@@ -62,6 +62,7 @@ Possible keys:
 | `type`                   |                                     |
 | `type.builtin`           |                                     |
 | `constructor`            |                                     |
+| `variant`                | Enum variants                       |
```

Comment:
Can you also add this to other files?

Reply:
I don't think it's necessary to add it to all the themes, each theme simply declares it's own set of keys it wants to style.

Reply:
Something like a todo comment with the key name on the theme files would make it easier to update them afterwards though

Reply:
Yes, just a TODO comment so that authors does not have to manually diff it themselves.